modules/pathtracing/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+ivw_module(pathtracing)
+
+set(HEADER_FILES
+    include/inviwo/pathtracing/pathtracingmodule.h
+    include/inviwo/pathtracing/pathtracingmoduledefine.h
+    include/inviwo/pathtracing/processors/dummyprocessor.h
+    include/inviwo/pathtracing/processors/volumepathtracer.h
+)
+ivw_group("Header Files" ${HEADER_FILES})
+
+set(SOURCE_FILES
+    src/pathtracingmodule.cpp
+    src/processors/dummyprocessor.cpp
+    src/processors/volumepathtracer.cpp
+)
+ivw_group("Source Files" ${SOURCE_FILES})
+
+# Add shaders
+set(SHADER_FILES
+    glsl/random.glsl
+    glsl/transmittancemethods.glsl #should really be called renderingmethods, its not strictly for transmittance
```

Comment:
None of the vector/matrix methods were used so there is no need for vectormatrixmethods.glsl?
If random.glsl is split up into files I think that they could go in a folder called random, but otherwise no need to move it.

I think that you can create the transmittance folder now.

Reply:
Only RayBBIntersection_TextureSpace is used. Ill keep random.glsl outside of util then

Skickat från Outlook för Android<https://aka.ms/AAb9ysg>